RESTSystem/API/Configuration/ConfigurationHandler.py

```diff
@@ -0,0 +1,56 @@
+""" Handler to serve the DIRAC configuration data
+"""
+
+__RCSID__ = "$Id$"
+
+import json
+
+from tornado import web, gen
+from RESTDIRAC.RESTSystem.Base.RESTHandler import WErr, RESTHandler
+from DIRAC import gConfig
+
+class ConfigurationHandler( RESTHandler ):
+
+  ROUTE = "/config"
+
+  @web.asynchronous
+  @gen.engine
+  def get( self ):
+    """ GET method to get configuration data
+    :return: requested data
+    """
+
+    args = self.request.arguments
+    if args.get( 'option' ):
+      path = args['option'][0]
+      result = yield self.threadTask( gConfig.getOption, path )
+      if not result['OK']:
+        raise WErr.fromError( result )
+      self.finish( json.dumps( result['Value'] ) )
```

Comment:
I don't think you need to do json.dumps. If I remember correctly if you just self.finish anything that's not a string it will be jsonified automatically and the proper HTTP header will be set.